### PR TITLE
Fix: camera reel screencapture hud hide UI issues

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

- locked shortcuts when in the screencapture mode
- fixed hide/show ui issues 
- refactored screencapture shortcuts to not have specific Controller for it

## How to test the changes?

1. Launch the explorer
2. Verify shortcuts for emotes and other features are not working when in screencapture mode
3. Verify show/hide UI doesn't make any issues
4. Verify show/hide screencapture HUD doesn't make issues

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by DafGreco — APPROVED** [review]
Changes verified on Windows and Macos! ✅

File: Explorer/Assets/Scripts/MVC/Manager/IMVCManager.cs [hunk 1]
```diff
@@ -25,6 +25,6 @@ public interface IMVCManager : IDisposable
 
         void SetAllViewsCanvasActive(bool isActive);
 
-        void SetAllViewsCanvasActiveExcept<T>(bool isActive) where T : IController;
+        void SetAllViewsCanvasActive(IController except, bool isActive);
```
Comment (lorenzo-ranciaffi) [inline Explorer/Assets/Scripts/MVC/Manager/IMVCManager.cs:28]: Wouldn't it be better if we could accept a collection of `IController`?

**Review by lorenzo-ranciaffi — APPROVED** [review]
LGTM - left a minor comment

File: Explorer/Assets/Scripts/MVC/Manager/IMVCManager.cs (hunk 1 above)
Comment (popuz) [reply Explorer/Assets/Scripts/MVC/Manager/IMVCManager.cs:28]: I would go with YAGNI and do it only when need comes

**Pull request merged**